lib/slack_interface/request_handler.coffee

```diff
@@ -18,8 +18,6 @@ class SlackInterfaceRequestHandler
             # We can store the user who issued the command
             @queuer = request.body['user_name']
             
```

Comment:
Perhaps create a check here to ignore any commands other than `unmute` if currently muted?